pkg/build/prometheus/metrics_via_collectors.go

```diff
@@ -0,0 +1,150 @@
+package prometheus
+
+import (
+	kselector "k8s.io/apimachinery/pkg/labels"
+	"strings"
+
+	"github.com/golang/glog"
+	buildapi "github.com/openshift/origin/pkg/build/apis/build"
+	internalversion "github.com/openshift/origin/pkg/build/generated/listers/build/internalversion"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	separator                          = "_"
+	BuildSubsystem                     = "openshift_build"
+	BuildCounter                       = "status_phase_terminal"
+	BuildCounterQuery                  = BuildSubsystem + separator + BuildCounter
+	BuildGauge                         = "status_phase_active"
+	BuildGagueQuery                    = BuildSubsystem + separator + BuildGauge
+	BuildHistogram                     = "status_duration"
```

Comment:
Sure ... and probably don't need `Histogram`, `Gauge`, or `Counter` in the variable names.  In further sampling of other usage within vendor tree, that seems to be a consistent aspect.  Might occasionally see something like `Count` as part of the name, but generally, without that, the variable name loses meaning.

Going to make general refurbish of the names.